hudi-flink-datasource/hudi-flink/src/test/resources/hive-site.xml

```diff
@@ -27,4 +27,9 @@
         <name>datanucleus.schema.autoCreateTables</name>
         <value>true</value>
     </property>
+
+    <property>
+        <name>some.important.prop</name>
+        <value>ololo</value>
```

Comment:
whatv is the purpose of this import?

Reply:
just to check in unit test that this property from hive-site.xml is transferred correctly to HiveTable config (`assertEquals("ololo", hiveTable.getParameters().get("hadoop.some.important.prop"));`)

Reply:
Don't think it is a sutable way, can we configue the `hadoop.some.important.prop` somewhere by using the code?

Reply:
Ok, used existing hive-site.xml property "hive.metastore.schema.verification" for this check.